### PR TITLE
[WIP]Autodeployment on Heroku

### LINKED ISSUE #3: Autodeploy project to Heroku
Autodeploy project to Heroku on each new merged commit.

### PR DESCRIPTION
**Pull request by S2606**
Fixes #3 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by jajodiaraghav** [comment]
Hi @S2606 To fix this, you can remove the `html` package from requirements file.

**Comment by S2606** [comment]
@jajodiaraghav I have rectified the issue and also without adding `apt` buildpack it runs fine. You can check it out 
https://badgeyay1.herokuapp.com/

**Comment by jajodiaraghav** [comment]
@S2606 I get internal server error when I click upload button.

File: app/requirements.txt [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 itsdangerous==0.24
 jinja2==2.9.6
 markupsafe==1.0
 werkzeug==0.12.2
 wtforms==2.1
-pyflakes==1.6.0
\ No newline at end of file
+pyflakes==1.6.0
+gunicorn
+setuptools
```
Comment (niccokunzmann) [inline app/requirements.txt]: usually, setuptools is not in the requirements.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+-r app/requirements.txt
```
Comment (niccokunzmann) [inline requirements.txt]: That is clever!

Comment (niccokunzmann) [reply requirements.txt]: Never saw this! I learned something new here :)

File: app.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "keywords": [
+        "badgeyay",
+        "fossasia",
+        "flask"
+    ],
+    "buildpacks": [
+        {
+            "url": "heroku/python"
+        }
+    ],
```
Comment (niccokunzmann) [inline app.json]: The comma in the end makes it invalid JSON. Codacy tells this.

**Comment by niccokunzmann** [comment]
@S2606 I made some comments, I can just merge it. What do you think?

File: app.json (hunk 3 above)
Comment (S2606) [reply app.json]: @niccokunzmann I will rectify it

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by S2606** [comment]
You'r welcome @niccokunzmann !

**Comment by jajodiaraghav** [comment]
@S2606 Can you please rename the deploy to `badgeyay-dev.herokuapp.com` or `fossasia-badgeyay.herokuapp.com`
This would make the url look a lot cleaner!

**Comment by S2606** [comment]
@jajodiaraghav https://badgeyay-dev.herokuapp.com/

@niccokunzmann Please change it to above link

**Pull request merged**